bucket/opera.json

```
@@ -5,32 +5,23 @@
     "license": "Freeware",
     "architecture": {
         "64bit": {
-            "url": "https://download1.operacdn.com/pub/opera/desktop/62.0.3331.43/win/Opera_62.0.3331.43_Setup_x64.exe",
+            "url": "https://download1.operacdn.com/pub/opera/desktop/62.0.3331.43/win/Opera_62.0.3331.43_Setup_x64.exe#/dl.7z",
             "hash": "eaf5b46adbe9b55ca04bf3b3f1d152df5a0f0d804001ebdd6645b976708a9c8f"
         },
         "32bit": {
-            "url": "https://download1.operacdn.com/pub/opera/desktop/62.0.3331.43/win/Opera_62.0.3331.43_Setup.exe",
+            "url": "https://download1.operacdn.com/pub/opera/desktop/62.0.3331.43/win/Opera_62.0.3331.43_Setup.exe#/dl.7z",
             "hash": "1f048841808933e155bbe63416de80b98d2a803232d43c5d3fbe5726692ff30c"
         }
     },
     "installer": {
-        "args": [
-            "/silent",
-            "/installfolder=\"$dir\"",
-            "/copyonly=1",
-            "/desktopshortcut=0",
-            "/startmenushortcut=0",
-            "/quicklaunchshortcut=0",
-            "/enable-stats=0",
-            "/enable-installer-stats=0",
-            "/launchbrowser=0"
-        ]
-    },
-    "uninstaller": {
-        "file": "launcher.exe",
-        "args": [
-            "/uninstall",
-            "/silent"
+        "script": [
+            "Remove-Item -Path \"$dir\\*_list\" -Force",
+            "Move-Item -Path \"$dir\\*\" -Destination \"$dir\\$version\" -Exclude 'Assets', 'launcher*', 'Resources.pri' -ErrorAction Ignore",
+            "if (!(Test-Path \"$dir\\$version\\localization\")) {",
+            "   New-Item -Path \"$dir\\$version\\localization\" -ItemType Directory | Out-Null",
+            "   Move-Item -Path \"$dir\\$version\\*.pak\" -Destination \"$dir\\$version\\localization\" -Exclude 'opera*' -ErrorAction Ignore",
+            "}",
+            "Set-Content -Path \"$dir\\installer_prefs.json\" -Value '{\"autoupdate\":false,\"enable_stats\":false,\"single_profile\":true}' -Encoding ASCII"
```

Maybe object would be better?
```suggestion
            "Set-Content -Path \"$dir\\installer_prefs.json\" -Value (@{ 'autoupdate'= $false; 'enable_stats' = $false; 'single_profile' = $true } | ConvertTo-Json) -Encoding ASCII"
```